tests/resources/VCH-Util.robot

```diff
@@ -371,9 +371,10 @@ Gather Logs From Test Server
     ${out}=  Run  curl -k -b vic-admin-cookies %{VIC-ADMIN}/container-logs.zip -o ${SUITE NAME}-%{VCH-NAME}-container-logs${name-suffix}.zip
     Log  ${out}
     Remove File  vic-admin-cookies
-    ${out}=  Run  govc datastore.download %{VCH-NAME}/vmware.log %{VCH-NAME}-vmware${name-suffix}.log
+    ${host}=  Get VM Host Name  %{VCH-NAME}
```

Comment:
These changes aren't part of https://github.com/vmware/vic/pull/6902 - where are they from?

Reply:
i have no idea why these are getting included, looking now

Reply:
It's a good idea to cherry-pick the commit from the other branch directly. I usually first merge to master, grab the commit from the master branch, apply it to the releases branch, then create the cherry-pick PR.

Reply:
ok, fixed!